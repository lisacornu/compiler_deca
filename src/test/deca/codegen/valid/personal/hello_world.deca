// Description:
//    test du sprint 1 : hello world
//
// Resultats:
//    Hello world !Hello world !
//
// Historique:
//    cree le 19/12/2024

{
    print("Hello world ! (avec print)") ;
    println("Hello world ! (avec println)") ;
}
