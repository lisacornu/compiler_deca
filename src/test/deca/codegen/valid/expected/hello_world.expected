Hello world ! (avec print)Hello world ! (avec println)
